tooling/reproducible/linux_repro_build_compare.sh

```diff
@@ -84,8 +84,116 @@ setEnvironment() {
   ls -ld "$CC" "$CXX" "/usr/lib/jvm/jdk-${BOOTJDK_VERSION}/bin/javac" || exit 1
 }
 
+# Function to check if a value is in the array
+containsElement () {
+  local e
+  for e in "${@:2}"; do
+    if [ "$e" = "$1" ]; then
+      return 0  # Match found
+    fi
+  done
+  return 1  # No match found
+}
+
+setBuildArgs() {
+  local CONFIG_ARGS=("--disable-warnings-as-errors" "--enable-dtrace" "--without-version-pre" "--without-version-opt" "--with-version-opt")
```

Comment:
We should be using the original build configure_args, not specifying these...?

Reply:
we need to fix the SBOM makejdk_any_platform_args which is losing the --configure-args "quotes"...

replace the code here: https://github.com/adoptium/temurin-build/blob/39a0b0094099f3aad0661211859c944fa0610c7d/makejdk-any-platform.sh#L54
with this, I "think" should fix that:
```
# Store params to this script as "buildinfo"
makeJdkArgs=""
for arg in "$@"; do
  # Quote the argument if it contains spaces
  if [[ "${arg}" =~ .*" ".* ]]; then
    makeJdkArgs="${makeJdkArgs} \"${arg}\""
  else
    makeJdkArgs="${makeJdkArgs} ${arg}"
  fi
done
echo "${makeJdkArgs}" > ./workspace/config/makejdk-any-platform.args
```

Reply:
yes just tested it, seems to do the right thing, you can then consume configure_args with this fix:
```
        {
          "name" : "makejdk_any_platform_args",
          "value" : "--clean-git-repo --jdk-boot-dir /home/jenkins/temurin-build/jdk-22 --configure-args \" --disable-warnings-as-errors --with-version-opt=LTS --with-version-pre=ea\" --target-file-name jdk-hotspot.tar.gz --release --clean-libs --tag jdk-23+24_adopt --create-sbom --user-openjdk-build-root-directory /home/jenkins/build2 --use-adoptium-devkit gcc-11.3.0-Centos7.6.1810-b03 --freetype-dir bundled --use-jep319-certs --create-debug-image --build-variant temurin jdk"
        },
```

Reply:
I'm not sure about this --configure-args. will this work for other cases? Like the value of it may be variable as `--disable-warnings-as-errors --with-version-opt=LTS --enable-dtrace --without-version-pre`?

Reply:
@sophia-guo i've created a PR to fix this, so we can get some builds you can test this with...
https://github.com/adoptium/temurin-build/pull/3835